src/osd/PG.cc

```diff
@@ -1237,10 +1237,11 @@ void PG::calc_replicated_acting(
   set<pg_shard_t> *acting_backfill,
   ostream &ss)
 {
-  ss << "calc_acting newest update on osd." << auth_log_shard->first
+  pg_shard_t auth_log_shard_id = auth_log_shard->first;
```

Comment:
what could be improved whit this local variable?

Reply:
Actually, the local variable `pg_shard_t auth_log_shard_id` is already existent before, I just change the location of its definition to reduce calling times of `auth_log_shard->first`. I think there will be a little bit improvement (although it can be ignored) to execution efficiency, or at least it makes the code looks more clean.